### PR TITLE
feat: add babel runtime option

### PR DESCRIPTION
**Pull request by jquense**
BREAKING CHANGE: runtime enabled by default for web-app targets

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -41,7 +41,9 @@ function preset(_, options = {}) {
     opts.include = ['proposal-object-rest-spread'];
 
     // in a web app assume we are using webpack to handle modules
+    // and we want the runtime
     if (target === 'web-app') {
+      opts.runtime = options.runtime == null ? false : options.runtime;
```
Comment (taion) [inline index.js]: how does this default it to `true` for `web-app`?

File: index.js [hunk 2]
```diff
@@ -41,7 +41,9 @@ function preset(_, options = {}) {
     opts.include = ['proposal-object-rest-spread'];
 
     // in a web app assume we are using webpack to handle modules
+    // and we want the runtime
     if (target === 'web-app') {
+      opts.runtime = options.runtime == null ? true : options.runtime;
```
Comment (taion) [inline index.js]: `defaultOptions.runtime` is `false`, though; is this an issue?

Comment (jquense) [reply index.js]: no `options` is the passed in value, `opts` the one with defaults applied

File: test/plugin.test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+describe('Preset', () => {
```
[the remark this answers is not in the document]
Comment (jquense) [reply test/plugin.test.js]: this is just quick smoke test here pre publish

**Comment by jquense** [comment]
ping

File: index.js (hunk 2 above)
Comment (taion) [reply index.js]: oh, can we rename this to `explicitOptions` or something then?

**Pull request merged**